dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -357,18 +462,23 @@ public void testFolderWithFileMove()
     Route route = Route.newBuilder().setDst(dstC.getAbsolutePath()).setSrc(c.getAbsolutePath())
         .setLength(length).build();
     Route route2 =
-        Route.newBuilder().setDst(b.getAbsolutePath()).setSrc(a.getAbsolutePath()).build();
-    Route route3 =
         Route.newBuilder().setDst(dstD.getAbsolutePath()).setSrc(d.getAbsolutePath()).build();
+    Route route3 =
+        Route.newBuilder().setDst(b.getAbsolutePath()).setSrc(a.getAbsolutePath()).build();
     routes.add(route);
     routes.add(route2);
-    routes.add(route3);
     WriteOptions writeOptions =
-        WriteOptions.newBuilder().setOverwrite(false).setCheckContent(true).build();
+        WriteOptions.newBuilder().setOverwrite(false).setCheckContent(true)
+            .setWriteType(WritePType.CACHE_THROUGH).build();
     UfsReadOptions read =
         UfsReadOptions.newBuilder().setUser("test").setTag("1").setPositionShort(false).build();
     ListenableFuture<List<RouteFailure>> move = mWorker.move(routes, read, writeOptions);
     List<RouteFailure> failures = move.get();
+    assertEquals(0, failures.size());
+    List<Route> routes2 = new ArrayList<>();
+    routes2.add(route3);
+    ListenableFuture<List<RouteFailure>> move2 = mWorker.move(routes2, read, writeOptions);
+    failures = move2.get();
```

Comment:
I'm still a little confused why we need to move twice. Do we know the root cause?